es-ds-docs/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "nuxt-app",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "build": "nuxt build",
+    "dev": "nuxt dev",
+    "generate": "nuxt generate",
+    "preview": "nuxt preview",
+    "postinstall": "nuxt prepare"
+  },
+  "dependencies": {
+    "@energysage/es-bs-base": "^2.0.4",
+    "@nuxtjs/google-fonts": "^3.2.0",
+    "nuxt": "^3.12.4",
+    "vue": "latest"
+  },
+  "devDependencies": {
+    "eslint-config-energysage": "^2.2.3",
```

Comment:
We might want to omit as this could be out-of-date with vue3

Reply:
Fair - removed. We'll have to figure out how to set up the linting we want.